index.html

```diff
@@ -734,12 +859,21 @@ <h3 id="reacting-to-revocation">
         </h3>
         <p>
           When the user agent learns that the user no longer intends to grant permission to use a
-          <a>feature</a>, <dfn>react to the user revoking permission</dfn> by running these steps:
+          feature described by the {{PermissionDescriptor}} |descriptor| in the context
+          described by the [=permission key=] |key|, <dfn>react to the user revoking permission</dfn>
+          by running these steps:
         </p>
         <ol class="algorithm">
           <li>
-            <a>Queue a global task</a> on the [=user interaction task source=] to run that
-            feature's [=powerful feature/permission revocation algorithm=].
+            <a>Queue a global task</a> on the [=user interaction task source=] to:.
```

Comment:
You didn't introduce this, but I missed the fact that [global tasks need to take a global object as a parameter](https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-global-task) to decide where to run. But there may not be a global when the user revokes permission: they can do it from the browser's settings with no pages open. And we probably don't need a task to remove permission store entries anyway.